lagrangian_planar/init_1d.H

```diff
@@ -34,12 +34,16 @@ write_model(std::string model_name,
             const Array2D<Real, 0, NPTS_MODEL-1, 0, model::nvar-1>& model_hse) {
 
     // Write the file
+    Real res = (xmax - xmin) / static_cast<Real>(nx);
```

Comment:
if you use
```std::string num_to_unitstring(const Real value)```
defined in `model_util.H`, doesn't it handle this for you?

Reply:
yes, it can handle it; to be honest I forgot its existence. Now is incorporated